src/Culling/babylon.boundingBox.ts

```diff
@@ -93,7 +102,7 @@
             max.addToRef(min, this.center).scaleInPlace(0.5);
             max.subtractToRef(min, this.extendSize).scaleInPlace(0.5);
 
-            this._update(worldMatrix || this._worldMatrix || Matrix.Identity());
+            this._update(worldMatrix || _identityMatrix, extraWorldExtent || 0);

```

Comment:
do not pass  `this._worldMatrix` anymore as we want this function to behave as the constructor.